src/System.Net.Http/src/System/Net/Http/Unix/HttpClientHandler.Unix.cs

```diff
@@ -39,8 +39,17 @@ public CookieContainer CookieContainer
 
         public ClientCertificateOption ClientCertificateOptions
         {
-            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
-            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            get
+            {
+                return ClientCertificateOption.Manual;
+            }
+            set
+            {
+                if (ClientCertificateOption.Manual != value)
+                {
+                    throw NotImplemented.ByDesignWithMessage("libcurl does not support automatic client certificate selection");
```

Comment:
As an aside, assuming we're keeping CurlHandler as a separate object, would it make more sense to have all of these properties just delegate to the corresponding properties on CurlHandler, and then have these PlatformNotSupported / NotImplementedExceptions in CurlHandler?